added confluence-cloud-content-get to Atlassian Confluence Cloud Inte… (#40623)
demisto/content: Packs/AtlassianConfluenceCloud/ReleaseNotes/1_1_4.md

#### Integrations

##### Atlassian Confluence Cloud

> - Added support for **confluence-cloud-content-get** command that retrieves a content item by content id.

Reviewer: ```suggestion
- Added the  **confluence-cloud-content-get** command to retrieve a content item.
```
